### PR TITLE
Update to Parser name used in Workbook

### PR DESCRIPTION
**Pull request by richlilly2004**
Fixes #

## Proposed Changes

  -
  -
  -

### CODE REVIEW

**Comment by richlilly2004** [comment]
> Fixes #
> 
> ## Proposed Changes

Fixed the parser name that's used in the Metaparser

**Comment by richlilly2004** [comment]
bump for @YaronFruchtmann

**Review by YaronFruchtmann — COMMENTED** [review]
The plan is to have for every schema-based-table an abstract, generic counterpart:
NetworkSessions: NetworkSessions_Generic
Dns:Dns_Generic

I understand that the previous name is already used in 2 workbooks. I suggest to update the workbooks. if necessary, for compatibility, it is also possible to add a function Network_MetaParser=(){NetworkSessions_Generic};

**Comment by richlilly2004** [comment]
Bump now that the Cisco normalizer got approved
https://github.com/Azure/Azure-Sentinel/pull/1964

File: Parsers/Normalized Schema - Networking (v1.0.0)/MetaParser/NetworkMetaParser.kql [hunk 1]
```diff
@@ -10,7 +10,7 @@
 // Reference: Using functions in Azure monitor log queries: https://docs.microsoft.com/azure/azure-monitor/log-query/functions
 // Reference: Sentinel normalization documentation: https://aka.ms/sentinelnormalizationdocs
 //
-// Alias name: NetworkSessions_Generic
+// Alias name: Network_MetaParser
```
[the remark this answers is not in the document]
Comment (YaronFruchtmann) [reply Parsers/Normalized Schema - Networking (v1.0.0)/MetaParser/NetworkMetaParser.kql]: Function names are documented here
https://docs.microsoft.com/en-us/azure/sentinel/dns-normalization-schema

**Pull request closed without merging**